app/api/helpers/humanize_helper.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+
+import humanize
+import pytz
+
+
+def humanize_helper(time):
+    """Returns time passed from now in a human readable duration"""
+
+    return humanize.naturaldelta(datetime.now(pytz.utc) 
+                - time.astimezone(pytz.utc)) + ' ago'
```

Comment:
Humanize adds ago automatically

Reply:
it adds ago with naturaltime, not with naturaldelta .

Reply:
You can use naturaltime with timedelta

`humanize.naturaltime(dt.timedelta(seconds=1001)) == 16 minutes ago`